docs/UserGuide.md

```diff
@@ -109,12 +117,17 @@ Example:
 
 Edits a food item from the displayed list of foods.
 
-Format: `edit INDEX [n/FOOD] [c/CALORIES] [t/MEAL_TYPE]`
+Format: `edit INDEX [n/FOOD_NAME] [c/CALORIES] [t/MEAL_TYPE]`
 
 * Edits a food item in the list displayed at the specified index.
 * The index refers to the index shown in the displayed food lists.
 * The index **must be a positive** integer.
 * At least one of the optional fields must be provided.
+* Only one of each optional fields can be provided.
```

Comment:
```suggestion

```
More than one optional fields can be provided actually. `edit 1 n/sushi t/dinner` is a valid command.